matlab/test/arrow/array/tTime64Array.m

```diff
@@ -128,10 +128,10 @@ function TestDuration(testCase, Unit)
         function TestValid(testCase, Unit)
             % Verify the Valid property returns the expected logical vector.
             times = seconds([100 200 NaN 355 NaN 400]);
-            arrray = testCase.ArrowArrayConstructorFcn(times, TImeUnit=Unit);
-            testCase.verifyEqual(arrray.Valid, [true; true; false; true; false; true]);
-            testCase.verifyEqual(toMATLAB(arrray), times');
-            testCase.verifyEqual(duration(arrray), times');
+            array = testCase.ArrowArrayConstructorFcn(times, TImeUnit=Unit);
```

Comment:
"TImeUnit" may be a typo too.

```diff
-TImeUnit
+TimeUnit
```

Reply:
Oh, probably. 

In some places I think I detected/added a rule to tolerate `TFoo`, and Ime is sometimes a form of IME (input method editor), but, you're right, that doesn't seem likely to apply here.